Sources/Seismography/GIRModule.swift

```diff
@@ -56,9 +48,9 @@ public final class GIRModule {
 
   public func dataType(name: String,
                        indices: GIRType? = nil,
-                       actions: (DataType) -> Void) -> DataType {
-    let data = DataType(name: name, indices: indices ?? TypeType.shared)
-    actions(data)
+                       category: Value.Category) -> DataType {
```

Comment:
What about constructors?

Reply:
We're not saving them in the metadata, I'm not sure why they need to be represented here.

Reply:
I need to know what constructors are present, in order, to IRGen values of these types.

Reply:
The signature maintains that for you.  Once I have type lowering working better you can use that to grab types too.

Reply:
Which signature?